cmake/toolchains/apple.toolchain.cmake

```diff
@@ -0,0 +1,926 @@
+# This file is part of the ios-cmake project. It was retrieved from
```

Comment:
I think you should add the Google copyright to the top of this file, too, and also create a entry for this in the `release_build_files/NOTICES` file.